docs/api.md

```diff
@@ -127,7 +127,7 @@ spec:
     rollingUpdate:
       maxSurge: 1
     type: RollingUpdate
-  version: v1.26.0
+  version: v1.26.4
```

Comment:
Why not to bump it to a minor release (1.27.0)?

Reply:
haven't had time to go through the changes in v1.27.0 and if any changes needed in our manifests etc, v1.27.0 is aimed for post kubecon.